src/library.tests/WorkloadInformationProviderTests.cs

```diff
@@ -33,13 +33,13 @@ public WorkloadInformationProviderTests()
         [InlineData(1, 100)]
         [InlineData(5, 20)]
         [InlineData(10, 3)]
-        [InlineData(1, 3)]
+        [InlineData(2, 3)]
         public async void GetReachableServicesAsync_HeadlessService(int numServices, int numAddresses)
         {
             ConfigureHeadlessService(numServices: numServices, namingFunction: (i) => $"myapp-{i}", numAddresses: numAddresses, addressHostNamingFunction: (i) => $"Host-{i}");
             var result = await _workloadInformationProvider.GetReachableEndpointsAsync(namespaceName: "", localProcessConfig: null, includeSameNamespaceServices: true, cancellationToken: default(CancellationToken));
             // Doing numServices-1 when calculating because we are adding empty subset for one service and that will be skipped
```

Comment:
Is this comment relevant now, because now dns name can be empty for headless services?